azure-functions/acbs-function/test-helpers/date-formats.js

```diff
@@ -0,0 +1,21 @@
+const invalidDateFormats = [
+  'yy-MM-dd',
+  'yy/MM/dd',
+  'yy MM dd',
+];
+
+const validDateFormats = [
```

Comment:
What's the point of these here? is it to just give all valid combinations? are all of them used? (sorry trying to get some context here!)

Reply:
I've put a load of different dates into each of the helpers to work out what moment accepts first, and will probably get rid of some when implementing behaviour if I'm pretty certain they're not used.
This is essentially painting the target around the arrow so I know what to aim for when ripping out moment

Reply:
Ah gotcha, sounds good in that case :)